target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -227,7 +227,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"1:lan" "2:lan" "3:lan" "4:lan" "5:lan" "0:wan" "6@eth0"
 		;;
-	kn_rc|\
+	kn_rc)
```

Comment:
Sorry for not noticing this earlier. Please merge the kn_rc into the 3g-6200n, ac1200pro, ai-br100... block which already uses ```"0:lan" "1:lan" "2:lan" "3:lan" "4:wan" "6@eth0"``` instead of duplicating existing code. Please test your changes afterwards, to make sure that it still does work as expected.